components/kyma-environment-broker/internal/metrics/step_result.go

```diff
@@ -121,7 +121,7 @@ func (c *StepResultCollector) OnOperationStepProcessed(ctx context.Context, ev i
 	}
 
 	switch {
-	case stepProcessed.Operation.Type == "provisioning":
+	case stepProcessed.Operation.Type == internal.OperationTypeProvision:
```

Comment:
potentially culprit why `compass_keb_provisioning_result` metrics disappeared recently. The old switch statement was comparing to `"provisioning"` but `internal.OperationTypeProvision` is defined as `provision` in
https://github.com/kyma-project/control-plane/blob/b40b0c560b1b1986acdd7ed657306a4b33982204/components/kyma-environment-broker/internal/model.go#L168

Reply:
likely similar to deprovisioning

Reply:
Nice catch!